config/locales/en.yml

```diff
@@ -2,9 +2,10 @@ en:
   privacy_notice:
     label: Privacy
   check_your_answers:
-    title: Check your answers before sending your application
+    title: Are you ready to send your application?
+    hint: Check your answers first
```

Comment:
I think that this will need to be added to the view as well before it has any effect.

Reply:
@leenagupte - this is beyond my knowledge :-/ Could someone else take a look please?

Reply:
Looking now 👀

Reply:
https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/167